kvdb-rocksdb/src/lib.rs

```diff
@@ -229,19 +231,42 @@ struct DBAndColumns {
 	column_names: Vec<String>,
 }
 
+fn static_property_or_warn(db: &DB, prop: &str) -> usize {
+	match db.property_int_value(prop) {
+		Ok(Some(v)) => v as usize,
+		_ => {
+			warn!("Cannot read expected static property of RocksDb database: {}", prop);
+			0
+		}
+	}
+}
+
+impl MallocSizeOf for DBAndColumns {
+	fn size_of(&self, ops: &mut parity_util_mem::MallocSizeOfOps) -> usize {
+		self.column_names.size_of(ops)
+			+ static_property_or_warn(&self.db, "rocksdb.estimate-table-readers-mem")
+			+ static_property_or_warn(&self.db, "rocksdb.cur-size-all-mem-tables")
```

Comment:
What do you think about including [`rocksdb.block-cache-usage`](https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h#L788) here?

Reply:
Added it, but we indeed probably need to think about sophisticated test with real database after this general approach to measurements (via parity-util-mem and db properties) is established

I'll add an issue for that